web/src/components/organisms/Dashboard/index.tsx

```diff
@@ -1,28 +1,128 @@
-import { useAuth } from "@reearth-cms/auth";
+import { PlusOutlined } from "@ant-design/icons";
+import styled from "@emotion/styled";
 import Button from "@reearth-cms/components/atoms/Button";
-import React from "react";
-import { useNavigate } from "react-router-dom";
+import MoleculeHeader from "@reearth-cms/components/molecules/Common/Header";
+import WorkspaceCreationModal from "@reearth-cms/components/molecules/Common/WorkspaceCreationModal";
+import WorkspaceMenu from "@reearth-cms/components/molecules/Common/WorkspaceMenu";
+import Search from "antd/lib/input/Search";
+import Layout, { Content, Header } from "antd/lib/layout/layout";
+import Sider from "antd/lib/layout/Sider";
+import React, { useState } from "react";
+import { useParams } from "react-router-dom";
 
 import useHooks from "./hooks";
 
 export type Props = {
   workspaceId?: string;
 };
 
-const Dashboard: React.FC<Props> = ({ workspaceId }) => {
-  const { logout } = useAuth();
-  const navigate = useNavigate();
-
-  const { user } = useHooks(workspaceId);
+const Dashboard: React.FC<Props> = () => {
```

Comment:
I don't think you need Props here. You aren't passing the workspaceId, you are getting it from useParams, which takes it from the url, right?